analyses/independent-samples/00-repeated-samples.Rmd

```diff
@@ -215,6 +216,7 @@ independents <- sample_df %>%
   filter(experimental_strategy == "WGS",
          sample_type == "Tumor", 
          composition != "Derived Cell Line",
+         pathology_diagnosis != "Metastatic secondary tumors"
```

Comment:
I am just noticing that there are instances of tumors having multiple primary diagnoses, for eg: `Ependymoma;Metastatic secondary tumors`, so we probably ought to do a grepl here

Reply:
Done.